pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -52,12 +52,17 @@ import (
 	"github.com/IBM/ibm-block-csi-operator/pkg/internal/ibmblockcsi"
 	kubeutil "github.com/IBM/ibm-block-csi-operator/pkg/util/kubernetes"
 	oversion "github.com/IBM/ibm-block-csi-operator/version"
+	"github.com/go-logr/logr"
```

Comment:
I think these changes are [already in develop](https://github.com/IBM/ibm-block-csi-operator/blob/3c6d21d87f618231937ae63bbd243ae694e0dbbc/pkg/controller/ibmblockcsi/ibmblockcsi_controller.go#L55)

Reply:
This PR is for 1.5.1, not develop.
I changed the base branch accordingly.